sopel/coretasks.py

```diff
@@ -698,14 +698,8 @@ def _remove_from_channel(bot, nick, channel):
 def _send_who(bot, channel):
     if 'WHOX' in bot.isupport:
         # WHOX syntax, see http://faerion.sourceforge.net/doc/irc/whox.var
-        # Needed for accounts in WHO replies. The random integer is a param
-        # to identify the reply as one from this command, because if someone
-        # else sent it, we have no fucking way to know what the format is.
```

Comment:
I wouldn't delete quite _that_ much of the comment. At least the idea of "hav[ing] no fucking way to know what the format is" if the request was sent by other code should be preserved somewhere.